azure-iot-provisioning-devicesdk/azure/iot/provisioning/devicesdk/transport/device_provisioning_constant.py

```diff
@@ -0,0 +1,23 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module contains constants related to the transport package.
+"""
+
+USER_AGENT = "azure-iot-provisioning-devicesdk" + "/" + "0.0.1"
+"""
+Default interval for polling, to use in case service doesn't provide it to us.
+"""
+POLLING_INTERVAL = 2000
```

Comment:
should this be `DEFAULT_POLLING_INTERVAL` since it's a default value, and for default timeout you've called it `DEFAULT_TIMEOUT`?